src/styles/todo.scss

```diff
@@ -8,7 +8,7 @@
   font-size: 24px;
   line-height: 1.4em;
   border-bottom: 1px solid #ededed;
-  
+
```

Comment:
Where do you use this file? Better move it to your component folder.

Reply:
todo.scss I use in file index.scss